### PR TITLE
Arcaflow sysbench cpu hog scenario

### PR DESCRIPTION
**Pull request by tsebastiani**
cpu hog scenario + basic arcaflow documentation.

### CODE REVIEW

File: scenarios/arcaflow/sysbench-cpu-hog/input.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+kubeconfig: {}
+node_selector:
+  kubernetes.io/hostname: minikube-m03
```
Comment (tsebastiani) [inline scenarios/arcaflow/sysbench-cpu-hog/input.yaml:3]: That is the bug that I'm waiting to fix in the Arca engine, kept here for debugging, will be removed before merging.

File: docs/arcaflow_scenarios.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            - scenarios/arcaflow/sysbench-cpu-hog/input.yaml
+```
+
+##### input.yaml
+The implemented scenarios can be found in *scenarios/arcaflow/<scenario_name>* folder.
+The entrypoint of each scenario is the *input.yaml* file. 
+In this file there are all the options to set up the scenario accordingly to the desired target 
+#### config.yaml
+The arcaflow config file. Here you can set the arcaflow deployer and the arcaflow log level.
+The supported deployers are:
```
Comment (paigerube14) [inline docs/arcaflow_scenarios.md:25]: I would just add a point here to be sure to install/configure whatever deployer type you select from the below list

File: docs/arcaflow_scenarios.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+kraken:
+    ...
+    chaos_scenarios:
+        - arcaflow_scenarios:
+            - scenarios/arcaflow/sysbench-cpu-hog/input.yaml
+```
+
+##### input.yaml
+The implemented scenarios can be found in *scenarios/arcaflow/<scenario_name>* folder.
+The entrypoint of each scenario is the *input.yaml* file.
```
Comment (paigerube14) [inline docs/arcaflow_scenarios.md:40]: can we add a link here to the input file you're adding as part of this pr even if just refernced as an example

Comment (tsebastiani) [reply docs/arcaflow_scenarios.md:40]: added, thank you.

**Pull request merged**